### PR TITLE
RWA-2750 Fix CVE-2023-20883 Fix

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-2750

### Change description ###
Fix CVE-2023-20883 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@
     <notes>Wait for spring boot 3.10 upgrades</notes>
     <cve>CVE-2023-20883</cve>
   </suppress>
-  <suppress until ="2023-08-31">
+  <suppress until ="2023-09-07">
```
Comment (raghera) [inline config/owasp/suppressions.xml:80]: Can put to the end of the next month if you want to e.g 30 Sept

**Review by raghera — APPROVED** [review]
Comments do not block this being merged so approved.

File: build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ plugins {
   id 'pmd'
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.1.0'
-  id 'org.springframework.boot' version '2.7.11'
+  id 'org.springframework.boot' version '2.7.12'
```
[the remark this answers is not in the document]
Comment (skandala-cgi) [reply build.gradle]: done

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (skandala-cgi) [reply config/owasp/suppressions.xml]: done

**Review by raghera — APPROVED** [review]

**Pull request merged**